chall05/estina.c

```diff
@@ -0,0 +1,47 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   estina.c                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: estina <estina@student.42madrid.com>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2020/04/16 14:07:38 by estina            #+#    #+#             */
+/*   Updated: 2020/04/16 14:59:57 by estina           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+static char	ft_lower(char c)
+{
+	if (c >= 'A' && c <= 'Z')
+		return (c + 32);
+	else
+		return (c);
+}
+
+static int	ft_check_c(char c, int type)
+{
+	if (c == 'c')
+		return (type ? 1 : 0);
+	return (type ? 0 : 1);
+
+}
+
+int			ft_ie_except_after_c(char* str)
+{
+	int		i;
+	int		rule;
+
+	if (!str || !*str)
+		return (0);
```

Comment:
42 teached me that I should care about this in functions also ;)

Reply:
If I remember well, the only functions that need to care about this, are the chained list functions, and there's a reason behind that, it's because `NULL` is considered as an empty list.  
Otherwise, if you take the standard library for example, try passing `NULL` to `strstr`, it will crash ;)